src/Propel/Generator/Platform/MysqlPlatform.php

```diff
@@ -53,19 +54,22 @@ class MysqlPlatform extends DefaultPlatform
     protected function initialize(): void
     {
         parent::initialize();
-        unset($this->schemaDomainMap[PropelTypes::UUID]);
         $this->setSchemaDomainMapping(new Domain(PropelTypes::BOOLEAN, 'TINYINT', 1));
         $this->setSchemaDomainMapping(new Domain(PropelTypes::NUMERIC, 'DECIMAL'));
         $this->setSchemaDomainMapping(new Domain(PropelTypes::LONGVARCHAR, 'TEXT'));
-        $this->setSchemaDomainMapping(new Domain(PropelTypes::BINARY, 'BLOB'));
-        $this->setSchemaDomainMapping(new Domain(PropelTypes::VARBINARY, 'MEDIUMBLOB'));
+        $this->setSchemaDomainMapping(new Domain(PropelTypes::BINARY, 'BINARY'));
+        $this->setSchemaDomainMapping(new Domain(PropelTypes::VARBINARY, 'VARBINARY'));
```

Comment:
Isn't this a BC break, in the case of `MEDIUMBLOB `=> `VARBINARY` because of size?

Reply:
Yes, that might be a problem for some users. But I don't see another way around it, those declarations are not accurate anymore, and we need them to be (this produced very hard to find errors).
Should be easy to fix though, they just have to change the type in their schema.xml. But yes, the whole thing is a bit unfortunate.

Reply:
Thanks for the explanation!
What happens if customers have already up and running DB with `VARBINARY => BLOB` and we'll switch to `VARBINARY => VARBINARY`.
As far as I can see from the code, there will be no migration applied and they will continue using `BLOB` as before in the DB.

Reply:
I wonder if we could seperate those two things, merging this PR with the UUID changes, and further setting the "cleanup" to a separate PR and discussion?
Maybe that helps to also make review easier.

Reply:
Hmm, I probably can undo it for the varbinary type. The binary type (currently mapping to blob) is central to working with UUIDs, I cannot just leave it out. Adding a mumbo jumbo alias is not a path a want to start heading into. Should I remove migration and tests for varbinary?

Reply:
Sounds fair

Reply:
Alright, I removed those changes again. I thought I was doing more with varbinary, but it was really just the mapping.